db/osqlblockproc.c

```diff
@@ -140,6 +146,34 @@ static int osql_bplog_key_cmp(void *usermem, int key1len, const void *key1,
     oplog_key_t *k2 = (oplog_key_t *)key2;
 #endif
 
```

Comment:
1) add optimization: if reordering is off, only check the seq.
2) make sure that the tunable is read only so we dont flip it on the fly.

Reply:
Actually i have a testcase where the tunable is flipped online. The db can handle it.